readalongs/align.py

```diff
@@ -518,11 +518,13 @@ def save_readalong(
     save_xml(tokenized_xml_path, align_results["tokenized"])
 
     _, audio_ext = os.path.splitext(audiofile)
+    audio_path = output_base + audio_ext
+    audio_format = audio_ext[1:]
     if audiosegment:
-        audio_path = output_base + ".wav"
-        audiosegment.export(audio_path, format="wav")
+        if audio_format in ["m4a", "aac"]:
+            audio_format = "ipod"
+        audiosegment.export(audio_path, format=audio_format)
```

Comment:
Boy, it's unfortunate we have to do this. I can imagine so many other cases that could require different values for `format`. I hope we're not opening a tricky can of worms here!

I think it can stay as is for now, but maybe watch out in the future, and choose a default format to use instead? We could also implement a fallback: if this export fails (we'd need to figure out how to test for that!), then fallback to a format known to be generally supported (like AAC, or maybe Ogg? https://www.makeuseof.com/tag/audio-file-format-right-needs/).

Reply:
I suspect this is where your unit test is failing on Xenial, which suggests if we want the code to be fairly portable to machines that are not all super recent, it might be worth trying my idea and seeing if that makes the unit tests pass on Xenial.